prov/verbs/src/fi_verbs.c

```diff
@@ -104,13 +104,15 @@ int vrb_sockaddr_len(struct sockaddr *addr)
 
 static int
 vrb_get_rdmacm_rai(const char *node, const char *service, uint64_t flags,
-		 const struct fi_info *hints, struct rdma_addrinfo **rai)
+		uint32_t addr_format, void *src_addr, size_t src_addrlen,
+		void *dest_addr, size_t dest_addrlen, struct rdma_addrinfo **rai)
 {
 	struct rdma_addrinfo rai_hints, *_rai;
 	struct rdma_addrinfo **cur, *next;
 	int ret;
 
-	ret = vrb_fi_to_rai(hints, flags, &rai_hints);
+        ret = vrb_set_rai(addr_format, src_addr, src_addrlen, dest_addr,
+				       dest_addrlen, flags, &rai_hints);
```

Comment:
The indentation is off here.  Please use tabs at the start of lines, not spaces, and align 'dest_addrlen' under 'addr_format'.

The same issue shows up in other places in the code.  Please look at the github difference to see the alignment issues.